test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoTransferSuite.java

```diff
@@ -224,14 +225,94 @@ public List<HapiSpec> getSpecsInSuite() {
+                                    counterAlias.set(
+                                            ByteString.copyFrom(
+                                                    asSolidityAddress(counterId.get())));
+                                    otherAccountId.set(registry.getAccountID("otheraccount"));
+                                }))
+                .when(
+                        cryptoTransfer(
+                                        (spec, b) ->
+                                                b.addTokenTransfers(
+                                                                TokenTransferList.newBuilder()
+                                                                        .setToken(nftId.get())
+                                                                        .addNftTransfers(
+                                                                                ocWith(
+                                                                                        accountId(
+                                                                                                partyAlias
+                                                                                                        .get()),
+                                                                                        accountId(
+                                                                                                counterAlias
+                                                                                                        .get()),
+                                                                                        1L)))
+                                                        .setTransfers(
+                                                                TransferList.newBuilder()
+                                                                        .addAccountAmounts(
+                                                                                aaWith(
+                                                                                        partyId
+                                                                                                .get(),
+                                                                                        +2))
+                                                                        .addAccountAmounts(
+                                                                                aaWith(
+                                                                                        otherAccountId
+                                                                                                .get(),
+                                                                                        -2))))
+                                .signedBy(DEFAULT_PAYER, PARTY, "otheraccount")
+                                .via(NFT_XFER))
+                .then(getTxnRecord(NFT_XFER).logged());
```

Comment:
Are there any assertions here?

Reply:
I think just validating its `SUCCESS`. If we don't add any `hasKnownStatus(..` it by default checks success